api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -140,7 +140,7 @@ DbUser syncComplianceTrainingStatusV2(DbUser user, Agent agent)
 
   Optional<DbUser> findUserWithAuthoritiesAndPageVisits(long userId);
 
-  DbUser updateRasLinkIdMeStatus(String idMeUserName);
+  DbUser updateIdentityStatus(String userName);
```

Comment:
Replaced updateRasLinkIdMeStatus with updateIdentityStatus to reflect that there is no use updating the ID.me access module.